.eslintrc

```diff
@@ -28,45 +28,70 @@
     "sinon": false,
   },
   "rules": {
-    "block-scoped-var": 0,
-    "camelcase": 0,
-    "comma-dangle": [2, "always-multiline"],
-    "comma-style": [2, "last"],
-    "curly": [2, "all"],
-    "dot-notation": [2, {"allowKeywords": true}],
-    "eqeqeq": [2, "allow-null"],
-    "guard-for-in": 0,
-    "max-len": [2, 80, 2, {ignoreComments: true}],
-    "new-cap": [2, {"capIsNewExceptions": ["Deferred"]}],
-    "no-bitwise": 2,
-    "no-caller": 2,
-    "no-cond-assign": [2, "except-parens"],
-    "no-debugger": 2,
-    "no-empty": 2,
-    "no-eval": 2,
-    "no-extend-native": 2,
-    "no-extra-parens": 0,
-    "no-extra-semi": 2,
-    "no-irregular-whitespace": 2,
-    "no-iterator": 2,
-    "no-loop-func": 0,
-    "no-multi-str": 2,
-    "no-new": 2,
-    "no-plusplus": 0,
-    "no-proto": 2,
-    "no-redeclare": 0,
-    "no-script-url": 2,
-    "no-sequences": 2,
-    "no-undef": 2,
-    "no-underscore-dangle": 0,
-    "no-unused-vars": 2,
-    "no-with": 2,
-    "quotes": [2, "single", "avoid-escape"],
-    "quote-props": [1, "consistent-as-needed"],
-    "semi": [2, "always"],
-    "space-infix-ops": 0,
-    "strict": [2, "never"],
-    "valid-typeof": 2,
-    "wrap-iife": [2, "inside"]
+    "block-scoped-var": "off",
```

Comment:
Yeah I noticed it in their docs and thought "I like that". It's much clearer as I always forget what the numbers mean.